WellCoffeeInventoryAPI/build.gradle

```diff
@@ -21,7 +21,14 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.security:spring-security-crypto'
+	implementation 'io.jsonwebtoken:jjwt-jackson:0.11.5'
```

Comment:
Are we using jwt or jsession?  Unused dependencies should be removed.

Reply:
jsession. Resolved